src/application/admin/service_test.go

```
@@ -0,0 +1,63 @@
+
+func TestService_ValidateToken(t *testing.T) {
+	mockJwt := new(mockjwt.JwtService)
+	svc := NewService(mockJwt)
+
+	t.Run("success", func(t *testing.T) {
+		exp := time.Now().Add(time.Hour)
+		mockJwt.On("ValidateToken", "token-abc").Return(&entity.JwtToken{
+			UserId:    "user-123",
+			ExpiredAt: exp,
+		}, nil)
+		resp, err := svc.ValidateToken("token-abc")
```

golangci-lint (1.64.8)

assignments should only be cuddled with other assignments

(wsl)